core/notify.py

```diff
@@ -105,6 +105,9 @@ def email(
     # Send all emails to the JML team
     email_addresses += settings.JML_TEAM_EMAILS
 
+    if settings.JML_ONLY_SEND_EMAILS_TO_JML_TEAM:
+        email_addresses = settings.JML_TEAM_EMAILS
+
```

Comment:
```suggestion
if settings.JML_ONLY_SEND_EMAILS_TO_JML_TEAM:
    email_addresses = settings.JML_TEAM_EMAILS
else:
    email_addresses += settings.JML_TEAM_EMAILS
```

May as well pull the above into an else

Reply:
Discussed offline.

I mildly prefer it without the `else` clause for the following reasons:
- easier to remove the code I've added
- the original flow hasn't changed and what I've added is going to be an exception to that
- the else clause puts the the 2 statements at the same indentation, which makes the only thing differentiating them `+=` vs `=`
- easier to add other code that might modify the `email_addresses`